Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java

```diff
@@ -57,7 +79,7 @@ public void setAdmins(Map<String, String> admins) {
 	@Override
 	public User findUser(String name) {
 		if (admins.containsKey(name)) {
-			String storedPW = admins.get(name);
+			String storedPW = String.valueOf(admins.get(name));
```

Comment:
@jdamerow  Actually I had defined the map as <String,Object> so thats why used String.valueOf to get the String value. Now I have changed the generics of Map as <String,String> so there is no need to use String.valueOf